cmd/sprint-automation/main.go

```diff
@@ -253,32 +256,32 @@ func usersOnCallAtTime(client *pagerduty.Client, slackClient *slack.Client, when
 		{
 			role:  roleTriagePrimary,
 			query: primaryOnCallQuery,
-			since: when.Add(-1 * time.Second),
-			until: when,
+			since: dayStart,
+			until: dayEnd,
 		},
 		{
 			role:  roleTriageSecondaryUS,
 			query: secondaryUSOnCallQuery,
-			since: when.Add(-24 * time.Hour),
-			until: when,
+			since: dayStart,
+			until: dayEnd,
 		},
 		{
 			role:  roleTriageSecondaryEU,
 			query: secondaryEUOnCallQuery,
-			since: when.Add(-24 * time.Hour),
-			until: when,
+			since: dayStart,
+			until: dayEnd,
 		},
 		{
 			role:  roleHelpdesk,
 			query: primaryOnCallQuery,
-			since: when.Add(-7 * 24 * time.Hour).Add(-1 * time.Second),
-			until: when.Add(-7 * 24 * time.Hour),
+			since: dayStart.Add(-7 * 24 * time.Hour),
+			until: dayStart.Add(-7 * 24 * time.Hour).Add(1 * time.Second),
```

Comment:
to keep the consistency:

```suggestion
			until: dayEnd.Add(-7 * 24 * time.Hour),
```